internal_event_handlers.go

```diff
@@ -335,6 +335,40 @@ func (wc *workflowEnvironmentImpl) RequestCancelTimer(timerID string) {
 	wc.logger.Debug("RequestCancelTimer", zap.String(tagTimerID, timerID))
 }
 
+func validateVersion(changeID string, version, minSupported, maxSupported Version) {
```

Comment:
instead of panic'ing here, panic inside GetVersion() if validateVersion() returns error. so it is very clear that method panics.